authentication/identity_providers/configuring-htpasswd-identity-provider.adoc

```diff
@@ -1,40 +1,39 @@
 :_content-type: ASSEMBLY
 [id="configuring-htpasswd-identity-provider"]
-= Configuring an HTPasswd identity provider
+= Configuring an htpasswd identity provider
 include::_attributes/common-attributes.adoc[]
 :context: configuring-htpasswd-identity-provider
 
 toc::[]
 
-[id="identity-provider-overview_{context}"]
-== About identity providers in {product-title}
+Configure the `htpasswd` identity provider to allow users to log in to {product-title} with credentials from an htpasswd file.
 
-By default, only a `kubeadmin` user exists on your cluster. To specify an
-identity provider, you must create a Custom Resource (CR) that describes
-that identity provider and add it to the cluster.
+To define an htpasswd identity provider, perform the following steps:
```

Comment:
I don't know if there is a hard-and-fast rule for introducing a list of procedures, but I tend to use _perform the followung tasks_ (or similar), because what follows aren't steps.